sqlglot/tokens.py

```diff
@@ -252,6 +252,7 @@ class TokenType(AutoName):
     QUALIFY = auto()
     QUOTE = auto()
     RANGE = auto()
+    RANGE_N = auto()
```

Comment:
why did you need to add this? i don't think it's needed

Reply:
you're right